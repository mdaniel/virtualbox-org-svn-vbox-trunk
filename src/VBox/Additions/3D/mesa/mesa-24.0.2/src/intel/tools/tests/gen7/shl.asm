shl(1)          a0<1>UW         a0<0,1,0>UW     0x0002UW        { align1 WE_all 1N };
shl(1)          g12.2<1>UD      g12.2<0,1,0>UD  0x0000000cUD    { align1 WE_all 1N };
shl(8)          g87<1>.xD       g4<0>.zD        0x00000004UD    { align16 1Q };
shl(8)          g13<1>.xyD      g5.4<0>.zwwwD   g6<0>.xyyyUD    { align16 1Q };
shl(8)          g114<1>UD       g26<4>.xUD      g27<4>.xUD      { align16 1Q };
shl(8)          g3<1>D          g2<0,1,0>D      0x00000003UD    { align1 1Q };
shl(16)         g3<1>D          g2<0,1,0>D      0x00000003UD    { align1 1H };
shl(8)          g37<1>D         g35<8,8,1>D     g5.6<0,1,0>UD   { align1 1Q };
shl(16)         g68<1>D         g23<8,8,1>D     g7.6<0,1,0>UD   { align1 1H };
shl(1)          a0<1>UD         g20<0,1,0>UD    0x00000008UD    { align1 WE_all 1N };
shl(8)          g59<1>.xUD      g58<4>.xUD      g57<4>.xUD      { align16 WE_all 1Q };
shl(8)          g38<1>UD        g38<8,8,1>UD    0x00000010UD    { align1 1Q };
shl(16)         g76<1>UD        g76<8,8,1>UD    0x00000010UD    { align1 1H };
