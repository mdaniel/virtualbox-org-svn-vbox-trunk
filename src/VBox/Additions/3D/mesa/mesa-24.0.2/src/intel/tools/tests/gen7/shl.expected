09 02 00 00 08 2d 00 22 00 02 00 00 02 00 02 00
09 02 00 00 21 0c 88 21 88 01 00 00 0c 00 00 00
09 01 60 00 a5 0c e1 2a 8a 00 0a 00 04 00 00 00
09 01 60 00 a5 04 a3 21 be 00 0f 00 c4 00 05 00
09 01 60 00 21 04 4f 2e 40 03 60 00 60 03 60 00
09 00 60 00 a5 0c 60 20 40 00 00 00 03 00 00 00
09 00 80 00 a5 0c 60 20 40 00 00 00 03 00 00 00
09 00 60 00 a5 04 a0 24 60 04 8d 00 b8 00 00 00
09 00 80 00 a5 04 80 28 e0 02 8d 00 f8 00 00 00
09 02 00 00 20 0c 00 22 80 02 00 00 08 00 00 00
09 03 60 00 21 04 61 27 40 07 60 00 20 07 60 00
09 00 60 00 21 0c c0 24 c0 04 8d 00 10 00 00 00
09 00 80 00 21 0c 80 29 80 09 8d 00 10 00 00 00
